examples/experimental/onnx/onnx_ptq_classification.py

```diff
@@ -0,0 +1,81 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+
+from typing import List
+
+import onnx
+
+from nncf.experimental.post_training.compression_builder import CompressionBuilder
+from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantization
+from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantizationParameters
+from nncf.experimental.onnx.dataloaders.imagenet_dataloader import create_dataloader_from_imagenet_torch_dataset
+
+
+def run(onnx_model_path: str, output_model_path: str,
+        dataset_path: str, batch_size: int, shuffle: bool, num_init_samples: int,
+        input_shape: List[int], ignored_scopes: List[str] = None):
+    print("Post-Training Quantization Parameters:")
+    print("  number of samples: ", num_init_samples)
+    print("  ignored_scopes: ", ignored_scopes)
+    onnx.checker.check_model(onnx_model_path)
+    original_model = onnx.load(onnx_model_path)
+    print(f"The model is loaded from {onnx_model_path}")
+
+    # Step 1: Initialize the data loader.
+    dataloader = create_dataloader_from_imagenet_torch_dataset(dataset_path, input_shape,
```

Comment:
I would make `create_dataloader_from_imagenet_torch_dataset` more generic and extend it with `mean`, `scale`, and `crop_ratio` setting them to the existing values by default, and also rename it to `create_imagenet_dataloader`

Reply:
Done